cmd/agent/subcommands/run/command.go

```diff
@@ -614,7 +615,7 @@ func startAgent(
 	log.Infof("Hostname is: %s", hostnameDetected)
 
 	// start remote configuration management
-	if pkgconfigsetup.IsRemoteConfigEnabled(pkgconfigsetup.Datadog()) {
+	if configUtils.IsRemoteConfigEnabled(pkgconfigsetup.Datadog()) {
```

Comment:
Please use components when possible, we receive the config comp in the this function.

```suggestion
	if configUtils.IsRemoteConfigEnabled(cfg) {
```